### DISCUSSION TITLE
How long it may take to generate SSL certs for custom domain?

### DISCUSSION THREAD

**Discussion by erolj** [beginner-corp/begin-community#112] [Support]
Hello there,
I'm stuck at **Step 2 of 3: Generating SSL certificates**, unable to finish the Domains setup. Meanwhile, the Production and Staging custom domain CNAME indicator shows green checkmarks.

Steps:
1. Go to 'Domains'
2. Click on 'Get Started'
3. Providing custom subdomain for the Production (prod.mydomain.com) and Staging (staging.mydomain.com) input fields.
4. In Step 2 of 3 CNAME indicators for both domains shows green checkmarks. 
5. At the bottom 'Start over' button is present. 

![firefox_QGKrFHbAc1](https://user-images.githubusercontent.com/26346481/173143292-98e0983e-eca9-40b1-89b7-68bfe57b2892.png)

Any advice on how long should I wait or how to proceed?

**Comment by brianleroux** [comment]
certs should be there if you see those check marks
